extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-hive-metastore-service-chart/values.yaml

```diff
@@ -15,11 +15,16 @@ image:
   dockerRepo: "ghcr.io/"
 
 mysql:
+  commonLabels:
+    aissemble-configuration-store: enabled
   enabled: true
   fullnameOverride: "hive-metastore-db"
   auth:
     # The schematool for the metastore service will create the database for us.
     database: "metastore"
+    # Note: Changing these values requires removal of the `hive-metastore-db-0` PVC, or manual modification of the
+    # persisted database.
+    username: $getConfigValue(groupName=hive;propertyName=HIVE_USERNAME)
```

Comment:
Should update the helm chart README with these new values

Reply:
updated!